[Gradle] Flag as a build-tool

adriens opened the pull request:
cf:

- #3955
- #4156

adriens committed:
Flag gradle as a build-tool

cf #3955

marcwrobel reviewed (changes requested):
`build-tool` must be added in `tag`, not in `iconSlug`

adriens committed:
Update gradle.md > Typo on build-tool and iconslug

cf https://github.com/endoflife-date/endoflife.date/pull/4157#pullrequestreview-1749414252

marcwrobel commented on products/gradle.md:
```diff
@@ -1,7 +1,7 @@
 title: Gradle
 category: app
-tags: java-runtime
+tags: java-runtime build-tool
```
@adriens, please keep the tags alphabetically ordered.

adriens replied:
Working on it : I didn't know it had to be : could it be checked by CI ?

adriens committed:
Update gradle.md > Kepp tags in alphabetical order

cf https://github.com/endoflife-date/endoflife.date/pull/4157#discussion_r1406673888

marcwrobel reviewed (commented):
LGTM, merging.
